### PR TITLE
Add pod highlighting

### PR DESCRIPTION
**Pull request by doy**

### CODE REVIEW

File: runtime/queries/pod/highlights.scm
Comment (the-mikedavis) [inline runtime/queries/pod/highlights.scm:1]: The `#match?` predicates here can be replaced with `#eq?`. `#eq?` is an exact match while `#match?` compiles the pattern as a regex

Comment (doy) [reply runtime/queries/pod/highlights.scm:1]: fixed

File: runtime/queries/pod/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  (sequence_letter) @letter
+  (#match? @letter "F")
+  (content) @markup.italic)
+(interior_sequence
+  (sequence_letter) @letter
+  (#match? @letter "I")
+  (content) @markup.italic)
+(interior_sequence
+  (sequence_letter) @letter
+  (#match? @letter "L")
```
Comment (pascalkuthe) [inline runtime/queries/pod/highlights.scm:60]: i think the same comment from @the-mikedavis should also be applied here

Comment (doy) [reply runtime/queries/pod/highlights.scm]: fixed

**Review by pascalkuthe — APPROVED** [review]

**Review by the-mikedavis — APPROVED** [review]

**Pull request merged**